lib/tasks/content_tagged_to_ancestors.rake

```diff
@@ -0,0 +1,11 @@
+namespace :content do
+  desc "Find all content that is tagged to a taxon and its direct ancestor"
```

Comment:
This task looks like it does more than finding, this might need updating?

Reply:
fixed